roles/base/tasks/main.yml

```diff
@@ -18,6 +39,7 @@
     - librbd-dev
     - lshw
   when: ansible_os_family == "Debian"
+  ignore_errors: yes
```

Comment:
Below is the error. Let me know if you any clues.
failed: [node1] => (item=unzip,vim-nox,curl,python-software-properties,git,mercurial,build-essential,perl,librbd-dev,lshw) => {"failed": true, "item": "unzip,vim-nox,curl,python-software-properties,git,mercurial,build-essential,perl,librbd-dev,lshw"}
msg: Could not fetch updated apt files